proiect/src/order/Order.java

```diff
@@ -0,0 +1,178 @@
+package order;
+
+
+import java.util.Date;
+
+import business.Business;
+import driver.Driver;
+import user.User;
+
+
+public class Order implements Comparable<Order> {
+	
+	private int id;
+	
+	private User client;
+	
+	private Business business;
+	
+	private Driver driver;
+		
+	private Date timePlacedOrder;
```

Comment:
maybe use LocalDate from java.time instead of Date from java.util package, it is the most used since Java 8

Reply:
Used LocalDateTime instead. LocalDate only holds day, month

Resolved in issue #5